src/blueprint/interfaces/blueprint-interface.ts

```diff
@@ -0,0 +1,81 @@
+
+
+
+export interface NamedSection {
+  markdownEntity: 'header'|'list'| 'special';
+  keyword: string | null;
+  identifier?: string; // Identifier a name of section. It is any non-empty combination of any character except [, ], (, ) and newline characters.
+  nestedSections?: NamedSection[] | NamedSection; // can be NamedSection or FormattedContent(see descendants descriptions)
+  description?: string;
+}
+
+export interface Group extends NamedSection {
+  markdownEntity: 'header';
+  keyword: string | 'Group';
```

Comment:
isn't 'Group' already included in type string?

Reply:
but on typehint it can suggest you to use that... plus in feature we may remove string from there, im still discovering blueprint spec